config/routes.rb

```diff
@@ -50,6 +50,7 @@
   get "/profile/password", to: 'users#edit_password'
   patch '/profile', to: 'users#update'
   put '/profile', to: 'users#update_password'
+  get "profile/orders", to: "orders#index"
```

Comment:
Still going to look into namespace/ resouces. Maybe wait until re-iteration at the end of the project.